# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/wgpu-core/build.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/wgpu-core/build.rs b/third_party/rust/wgpu-core/build.rs
rename from gfx/wgpu/wgpu-core/build.rs
rename to third_party/rust/wgpu-core/build.rs
--- a/gfx/wgpu/wgpu-core/build.rs
+++ b/third_party/rust/wgpu-core/build.rs
@@ -1,20 +1,16 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 fn main() {
     // Setup cfg aliases
     cfg_aliases::cfg_aliases! {
         // Vendors/systems
         wasm: { target_arch = "wasm32" },
         apple: { any(target_os = "ios", target_os = "macos") },
         unix_wo_apple: {all(unix, not(apple))},
 
         // Backends
         vulkan: { all(not(wasm), any(windows, unix_wo_apple)) },
         metal: { all(not(wasm), apple) },
         dx12: { all(not(wasm), windows) },
-        dx11: { all(not(wasm), windows) },
-        gl: { false },
+        dx11: { all(false, not(wasm), windows) },
+        gl: { all(not(wasm), unix_wo_apple) },
     }
 }